ios/browser/api/query_filter/query_filter_utils.mm

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/ios/browser/api/query_filter/query_filter_utils.h"
+#include "base/strings/sys_string_conversions.h"
+#include "brave/components/query_filter/utils.h"
+#import "net/base/mac/url_conversions.h"
+#include "url/gurl.h"
+
+#if !defined(__has_feature) || !__has_feature(objc_arc)
+#error "This file requires ARC support."
+#endif
+
+// MARK: - Implementation
+
+@implementation NSURL (QueryFilterUtilities)
+
+- (nullable NSURL*)
+    brave_stripTrackerParamsUsingInitiatorURL:(/*nullable*/ NSURL*)initiatorURL
+                            redirectSourceURL:
+                                (/*nullable*/ NSURL*)redirectSourceURL
+                                requestMethod:(NSString*)requestMethod
+                           isInternalRedirect:(BOOL)isInternalRedirect {
+  // Create new fake gurls because thats what
+  // `query_filter::MaybeApplyQueryStringFilter` expects
+  GURL initiator_url;
```

Comment:
you seem to have a mix of naming conventions here. In obj-c code follow obj-c naming conventions https://chromium.googlesource.com/chromium/src/+/main/styleguide/objective-c/objective-c.md#objective_c_style-matches-the-language
https://google.github.io/styleguide/objcguide.html#naming

Reply:
oh ok, ill do that. I was just trying to deal a girl version of the URL since the name initiatorURL was taken. I guess ill have to call it `initiatorGURL` then.

Reply:
Done